Meadowlark-js/backends/meadowlark-elasticsearch-backend/src/repository/QueryElasticsearch.ts

```diff
@@ -15,11 +14,7 @@ const moduleName = 'elasticsearch.repository.QueryElasticsearch';
  * Returns ElasticSearch index name from the given ResourceInfo.
  */
 export function indexFromResourceInfo(resourceInfo: ResourceInfo): string {
-  const adjustedResourceName = resourceInfo.isDescriptor
-    ? normalizeDescriptorSuffix(resourceInfo.resourceName)
```

Comment:
Not part of ticket, but removed normalizeDescriptorSuffix references (and thus metaed-core dependency) in backends. meadowlark-core now provides the normalized version.